django_password_validators/password_history/password_validation.py

```diff
@@ -12,11 +12,29 @@
 
 class UniquePasswordsValidator(object):
     """
-    Validate whether the password was once used by the user.
-
+    Validate whether the password was once used by the user
+    in the configured lookup_range.
     The password is only checked for an existing user.
     """
 
+    def __init__(self, lookup_range=float('inf')):
+        try:
+            self.lookup_range = (
+                lookup_range if lookup_range >= 0 else float('inf')
```

Comment:
@spokerman12, array indices and slices (which is where this is used later in the code) expect an integer and the default value of `float('inf')` will throw errors, no?

Reply:
@spokerman12, I checked and verified the code execution and realized this.

Whenever the `lookup_value` is infinity, it will never go to the `else` block, which is where the `lookup_value` is used to calculate the array index. So that is why this works okay.

We can resolve this comment thread now. 👍🏼